packages/portal/src/mixins/klaro.js

```diff
@@ -18,14 +23,13 @@ export default {
   },
 
   mounted() {
-    if (!this.klaro) {
-      this.klaro = window.klaro;
-    }
+    const script = document.createElement('script');
+    script.setAttribute('src', `https://cdn.jsdelivr.net/npm/klaro@${version}/dist/klaro-no-css.js`);
+    script.setAttribute('defer', true);
 
-    // If Matomo plugin is installed, wait for Matomo to load, but still render
-    // Klaro if it fails to.
-    const renderKlaroAfter = this.$waitForMatomo ? this.$waitForMatomo() : Promise.resolve();
-    renderKlaroAfter.catch(() => {}).finally(this.renderKlaro);
+    script.addEventListener('load', this.onKlaroScriptLoad);
+
+    document.head.appendChild(script);
```

Comment:
It's possible to use the `head` method in components too. So we could set the script in there instead.